src/Microsoft.Fast.Components.FluentUI/Components/Base/FluentInputBase.cs

```diff
@@ -109,6 +109,8 @@ public abstract partial class FluentInputBase<TValue> : FluentComponentBase, IDi
     /// </summary>
     protected internal FieldIdentifier FieldIdentifier { get; set; }
 
+    public bool FieldBound => ValueExpression != null || ValueChanged.HasDelegate;
```

Comment:
Could you set this property to `private`

Reply:
Done!